package.json

```diff
@@ -15,7 +15,6 @@
     "pack": "npm run build && electron-builder",
     "release": "npm run pack -- -mwl --publish=always",
     "release-ci": "npm run pack -- -mwl --publish=never",
-    "fix-sharp": "node node_modules/sharp/install/libvips && node node_modules/sharp/install/dll-copy",
```

Comment:
We havent been using this? Usually if we dont do this, then the compiled version would be done for different envs?

Reply:
See https://github.com/lovell/sharp/issues/3750

They have overhauled their install, the script we used for this step is no longer there. Seems like it's more similar to prebuildify to add `libvips` now, but if this doesn't work we will have to revisit and figure out how to get this working again.

Reply:
Nice, yeah thanks for the link. I remember doing something similar for synthetics